### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by sedroche**

### CODE REVIEW

File: walkthroughs/writing-walkthroughs/walkthrough.adoc [hunk 1]
```diff
@@ -443,7 +443,7 @@ Other attributes that are available to Walkthrough authors are:
 . Add another step to the numbered list in your Procedure:
 +
 ----
-. Click on link:\{ openshift-url }[Openshift Dashboard]
+. Click on link:\{openshift-url}[Openshift Dashboard]
```
Comment (aidenkeating) [inline walkthroughs/writing-walkthroughs/walkthrough.adoc]: Just tested this out, it looks like we can remove the `\` here too and it still gives the expected result, but the user won't have to manually remove the `\` anymore.

@finp Does removing the `\` seem fine to you? So long as what the user sees is

```
. Click on link:{openshift-url}[Openshift Dashboard]
```

and not

```
. Click on link:https://...[Openshift Dashboard]
```

Comment (pwright) [reply walkthroughs/writing-walkthroughs/walkthrough.adoc]: yep,  that seems correct.
It's only when you set subs="attributes" that the url will be rendered, depending on the way asciidoctor.js is called (which is a different conversation)

Comment (sedroche) [reply walkthroughs/writing-walkthroughs/walkthrough.adoc]: Ok, I'll remove the `\`

**Comment by sedroche** [comment]
@aidenkeating updated

**Review by aidenkeating — APPROVED** [review]

**Pull request merged**